lib/Makefile.am

```diff
@@ -16,7 +16,7 @@
 #  See the License for the specific language governing permissions and
 #  limitations under the License.
 
-SUBDIRS = ts records tsconfig bindings cppapi
+SUBDIRS = ts records . tsconfig bindings cppapi
```

Comment:
This is not needed.  I tested the build without having this in and it worked.  Approving for now to get the build fixed.

Reply:
Yeah, wasn't sure about that. The point of it was to force building the luajit before things that depend on it.

Reply:
I think this may be needed for `make check`.

Reply:
From what I recall, we attempted to fix the link issue first by having the lua code build first, but that was not the problem. 

This change may be neither sufficient nor necessary to fix the build.